src/style.css

```diff
@@ -1,3 +1,87 @@
-body {
+body,
+html {
   margin: 0;
+  padding: 0;
+  font-family: Roboto, sans-serif;
+  box-sizing: border-box;
+}
+
+ul {
+  list-style: none;
+  margin: 0;
+  padding: 0;
+  font-size: 0  ;
+}
+
+li {
+  position: relative;
+  margin: 0 10px;
+  display: inline-block;
+  /* background-color: #f28; */
+  font-size: 12px;
+}
+
+.nav__list {
+  text-transform: uppercase;
+
+}
+
+a.nav__list {
+  display: inline-block;
+  text-decoration: none;
+  color: #000;
+  padding: 23px 0;
+}
+
+a:hover {
+  color: #00acdc;
+}
+
+a.is-active {
+  color: #00acdc;
```

Comment:
It is a good practice to make all colors [variable](https://developer.mozilla.org/en-US/docs/Web/CSS/Using_CSS_custom_properties)